trin-core/src/jsonrpc/service.rs

```diff
@@ -253,8 +253,10 @@ fn serve_ipc_client(
                     .to_string()
                     .into_bytes(),
                 };
-                tx.write_all(&formatted_response).unwrap();
-                tx.flush().unwrap();
+                match tx.write_all(&formatted_response) {
+                    Ok(_) => tx.flush().unwrap(),
+                    Err(msg) => warn!("Unable to write bytes to unix stream, your request probably timed out: {:?}", msg),
```

Comment:
nit, to be clear that it's not a trin timeout but a caller (web3.py) timeout
```suggestion
                    Err(msg) => warn!("Unable to write bytes to unix stream, the requester has likely timed out: {:?}", msg),
```